src/main/java/lotto/domain/Prize.java

```diff
@@ -3,21 +3,20 @@
 import java.util.Arrays;
 
 public enum Prize {
-    FIFTH(3, 5000, "3개 일치"),
-    FOURTH(4, 50000, "4개 일치"),
-    THIRD(5, 1500000, "5개 일치"),
-    SECOND(5, 30000000, "5개 일치, 보너스 볼 일치"),
-    FIRST(6, 2000000000, "6개 일치");
+    FIFTH(3, 5000),
+    FOURTH(4, 50000),
+    THIRD(5, 1500000),
+    SECOND(5, 30000000),
+    FIRST(6, 2000000000);
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000);
```
위와 같이 쓰면 가독성이 향상 될 수 있어요 ㅎ